### ISSUE TITLE
Pretrained models

### ISSUE THREAD

**Issue by ZeWang95** [cyvius96/few-shot-meta-baseline#1]
Do you plan to release the pretained models?
Not everyone can afford pretraining on 8 GPUs.
Thanks.

**Comment by cyvius96** [comment]
Sure, I can upload the pretrained models later.

While not having enough GPUs, you can try to train the model with less GPUs. If it does not fit in the memory, it may be worth a try to train the model with smaller batch size, or with gradient accumulation and keep the batch size.

**Comment by cyvius96** [comment]
You can now download the pretrained models: https://www.dropbox.com/sh/ef2sm8d8qadhg3a/AAAIBotzaCKIdN1dJTvgDk-wa?dl=0
